src/pages/account/account-page.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { ReactElement } from "react";
```

Comment:
Please do the account page like it is in the mock-ups on Lucidchart.
- Centered on the page
- A description of the input field on the left-hand side
- The input field on the right-hand side